wandb/sdk/data_types/video.py

```diff
@@ -84,12 +87,11 @@ def __init__(
         self,
         data_or_path: Union["np.ndarray", str, "TextIO", "BytesIO"],
         caption: Optional[str] = None,
-        fps: int = 4,
+        fps: Optional[int] = None,
         format: Optional[str] = None,
     ):
         super().__init__()
 
-        self._fps = fps
```

Comment:
is this not used anywhere?

Reply:
ah looks like it

Reply:
Previously it was passed to the `encode` function.

Because this was only used in there, I figured it was best to remove it from a class variable and instead use it directly in the encode method to clarify the usage.

Reply:
yeah, i saw it is not used, good call to remove it